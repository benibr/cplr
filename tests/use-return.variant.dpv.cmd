../bin/cplr 'ret = 69' -dpv
