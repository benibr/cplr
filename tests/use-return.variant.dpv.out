Preparation phase
Generation phase
Generating code
================================================================================
     1	/* main */
     2	int main(int argc, char **argv) {
     3	    int ret = 0;
     4	/* statements */
     5	    ret = 69;
     6	/* done */
     7	    return ret;
     8	}
================================================================================
Generated: 187 bytes code, 122 bytes dump
Compilation phase
Compilation succeeded
Execution phase
Execution finished (ret=69).
